svgdigitizer/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.11.0"
```

Comment:
I don't think this is the recommended way to this anymore since https://peps.python.org/pep-0396/ has been rejected.

You can do something like the following to determine the version number:
```
>>> from importlib.metadata import version
>>> version('package name')
```